README.md

```diff
@@ -152,10 +173,56 @@ dashboard. On your development environment,
 make sure to use test keys. If you don't, you're
 going to be charged real money!
 
-#### Testing
+### Sripe keys setup
+
+#### Get stripe keys:
+
+Go to [stripe](https://stripe.com/en-br), create an account or just log in with you already have one. Acess the stripe dashboard and copy both publishable and secret keys.
+> make sure to use test keys. If you don't, you're
+going to be charged real money!
+
+![Screenshot from 2022-02-24 13-28-18](https://user-images.githubusercontent.com/44117439/155566057-97ab2370-7ed1-43ab-91fc-cdd4cb340390.png)
+ 
+ #### Configure the .env file:
+ 
+ Then after retrieving both keys copy them into your .env file on these lines:
+ ```
+ export STRIPE_API_KEY='REPLACE' # use your test private key from your stripe account
+ export STRIPE_API_PUBLIC='REPLACE' # use your test public key from your stripe account
+ ```
+
+### Start project:
+
+
+```bash
+bin/rails server
+```
+
+You can connect to your server at http://localhost:5000
+
+
+### Super admin
+
+There is a way to set your user as a super_admin. This role lets you access any of the nonprofits
+on your Houdini instance. Additionally, it gives you access to the super admin control panel to search all supporters and
+nonprofits, which is located at `/admin` url.
+  
+To create the super user, go to the rails console by calling:
+
+`bin/rails console`
+
+In the console, run the following:
+
+```ruby
+admin=User.find(1) #or the id of the user you want to add the role
+role=Role.create(user:admin,name: "super_admin")
```

Comment:
This should mention how you can create a super user as part of the nonprofit create command: `bin/rails houdini:nonprofit:create` but that, alternatively, you can do this if the user has already been created.

Reply:
This should still be modified to include the information about creating a super admin as part of the `bin/rails houdini:nonprofit:create` command.